mailer.js

```diff
@@ -292,8 +295,8 @@ module.exports = function (log) {
       template: 'postVerifyEmail',
       templateValues: {
         link: link,
-        androidUrl: this.androidUrl,
-        iosUrl: this.iosUrl,
+        androidUrl: this.androidUrl + postVerifyUtmParams,
+        iosUrl: this.iosUrl + postVerifyUtmParams,
```

Comment:
This'll break if we ever configure one of the URLs to have existing query parameters, may be better off using a URL joining library.  But we can do that in a follow-up.